specs/config/sample.json

```diff
@@ -0,0 +1,47 @@
+{
+  "default_category_option_combos": "HllvX50cXC0",
```

Comment:
Just as reminder, as mentioned on Slack, we probably want to have this inside specific data elements defs

Reply:
thanks, just removed it.